tox.ini

```diff
@@ -17,7 +17,7 @@ envlist =
 [testenv]
 skip_install = true
 deps =
-    setuptools < 74
+    setuptools  <74
```

Comment:
Should we keep the blank?

Reply:
I've cleaned up the spacing in zope.meta and applied it here.